playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -192,7 +192,7 @@
     when: origin_reconcile_bindings | bool or ent_reconcile_bindings | bool
     register: reconcile_bindings_result
     changed_when:
-    - reconcile_bindings_result.stdout.length > 0
+    - reconcile_bindings_result.stdout != ''
```

Comment:
@ewolinetz I meant something like this:
```
changed_when:
- reconcile_bindings_results.stdout
```
Then empty string would evaluate to False

or if it is possible that stdout is not present:
```
changed_when:
- reconcile_bindings_result.stdout | default(false, boolean=true)
```